frontend/frontend/src/components/Game.vue

```diff
@@ -106,10 +111,12 @@ export default {
 			this.timerState = "stopped";
 		},
 		async stopGame(gameWon = false) {
+			console.log(this.cards);
```

Comment:
do you mind removing the different console.log() scattered in the code. It's important we clean these up before pushing

Reply:
done